test/verify/check-connection

```diff
@@ -634,11 +634,17 @@ class TestConnection(MachineCase):
         # upgrade from distro version (our images have cockpit-ws preinstalled) sets up dynamic motd/issue symlink
         self.assertIn('Activate the web console', m.execute("cat /etc/motd.d/cockpit"))
         self.assertIn('Activate the web console', m.execute("cat /etc/issue.d/cockpit.issue"))
+        self.assertIn('root', m.execute("cat /etc/cockpit/disallowed-users"))
```

Comment:
The comment at the top of this paragraph talks about upgrade; but this tests the "clean install" scenario, can you please add a comment? Also, I think that existing comment is actually wrong, no? "cockpit-ws preinstalled" only applies to the -distropkg scenario, in all other scenarios we rpm -e all cockpit packages first.

Reply:
Right, this test is not executed on any `distropkg` scenario, the whole class has a `@skipDistroPackage` decorator.

A prepared image has erased the original image's cockpit and installed our build version, so yes, this is a clean install and I'll update the comment.
```
[root@fedora-37-127-0-0-2-2201 ~]# rpm -qa | grep cockpit
cockpit-bridge-278.33.g16ad2b4d6.dirty-1.fc37.x86_64
cockpit-system-278.33.g16ad2b4d6.dirty-1.fc37.noarch
cockpit-storaged-278.33.g16ad2b4d6.dirty-1.fc37.noarch
cockpit-networkmanager-278.33.g16ad2b4d6.dirty-1.fc37.noarch
cockpit-selinux-278.33.g16ad2b4d6.dirty-1.fc37.noarch
cockpit-packagekit-278.33.g16ad2b4d6.dirty-1.fc37.noarch
cockpit-pcp-278.33.g16ad2b4d6.dirty-1.fc37.x86_64
cockpit-ws-278.33.g16ad2b4d6.dirty-1.fc37.x86_64
```